applications/app/views/offlinePage.scala.html

```diff
@@ -14,10 +17,65 @@
     @* get the stylesheets downloading ASAP *@
     @fragments.stylesheets(None)
 
-    @* load the app *@
+    @* We don't use the inlineJSBlocking template because we need to specify
+    our own paths *@
```

Comment:
@rich-nguyen After talking to @sndrs about this, I think we should try to avoid duplication. I actually created a system to do this beforehand—you could pass in a flag to the template called `alwaysUseBundlesInDev` (or something), but that was quite complicated and too awkward.

We could either go back to a system such as this, or allow `window.curl.paths` to be pre-defined, e.g.:

``` js
// in offline page JS:
window.curlPaths = {
  // bundles
};

// in general JS:
window.curl = {
  api: 'require',
  paths: window.curlPaths || {
    // normal paths
  }
};
```

Can we get a consensus on this before I do the work, so I don't repeat myself? :smile: